examples/adb-lakehouse/versions.tf

```diff
@@ -4,5 +4,13 @@ terraform {
       source  = "hashicorp/azurerm"
       version = "~>3.13.0"
```

Comment:
Upgrade azurerm version as well. so we can create access connector without azapi provider: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_access_connector